ONSdigital/census-rm-action-service
# Image location change

# Motivation and Context
Image name is not consistent across travis, pom files etc...

# What has changed
build & push image as "eu.gcr.io/census-rm-ci/rm/census-rm-actionsvc"

# How to test?
**1.** check out branch

**2.** create a temporary commit by adding the following to .travis.yml file (before "- bash <(curl -s https://codecov.io/bash)" line) in "after_success:"  - 
```
- if [ "$TRAVIS_BRANCH" == "image-location-change" ] ; then
  docker login -u "${DOCKER_GCP_USERNAME}" -p "${DOCKER_GCP_PASSWORD}" "${DOCKER_GCP_REGISTRY}";
  docker tag "eu.gcr.io/census-rm-ci/rm/census-rm-actionsvc" "eu.gcr.io/census-rm-ci/rm/census-rm-actionsvc:test";
  docker push "eu.gcr.io/census-rm-ci/rm/census-rm-actionsvc:test";
  fi
```
and replace "branches" section with -
```
branches:
  only:
  - master
  - image-location-change
```
**3.** push commit to trigger a travis build

**4.** check "census-rm-ci" registry for tagged image

**5.** delete tagged image

**6.** rollback temporary commit changes
